libsolidity/inlineasm/AsmAnalysis.cpp

```diff
@@ -49,6 +49,22 @@ bool AsmAnalyzer::analyze(Block const& _block)
 	if (!(ScopeFiller(m_info, m_errorReporter))(_block))
 		return false;
 
+	/// Populate top level block with builtin functions
+	if (m_julia)
+	{
+		Scope& rootScope = scope(&_block);
+		rootScope.registerFunction("abort", {}, {});
```

Comment:
We might want to write a small parsing function for something like `keccak256(u256, u256) -> u256`, then it is also much more readable.